client.js

```diff
@@ -813,6 +855,18 @@ PeerStore.prototype.add = function (key, peer) {
   if (++this.used > this.max) this._evict()
 }
 
+PeerStore.prototype.remove = function (key, peer) {
+  var peers = this.peers.peek(key)
+  if (!peers) return false
+  var removed = peers.map.remove(peer.toString('hex'))
+  if (!removed) return false
+  this.used--
+  swap(peers.values, peers.values.length - 1, removed.index)
+  peers.values.pop()
+  if (!peers.values.length) this.peers.remove(key)
```

Comment:
Code looks similar to `_evict` below it, is there a chance for reuse?

Reply:
I ended up writing a new simple module that encapsulates this instead. https://github.com/mafintosh/record-cache. Wanna use this here and get rid of peer store in general